### DISCUSSION TITLE
Compare Staging vs. Production web site Example?

### DISCUSSION THREAD

**Discussion by openaudible** [Visual-Regression-Tracker/Visual-Regression-Tracker#433] [Q&A]
What is a simple way to visually compare a staging web site to a production web site.  For instance:

```
    String production = "https://website.com";
    String staging = "http://localhost:8081";
    String pages[] = {"/", "/about", "/contact", "/support", "/docs", "/purchase"};
   capture(production, pages);
   capture(staging, pages);
```
My site is just a basic web site. I don't have CI. Don't need a big history. Just want to be able to run a comparison between production and staging (what I'm about to push to production.)

When I make a change to html/css, I'd like to be able to tell what has changed (for desktop and mobile browsers.)

I have VRT running in docker and have successfully uploaded some images. But I'm not seeing how to add a second set of images to compare to the production images.

I have some of the java sample code running, and have uploaded images to the project, I'm not clear how to upload one as a production page and another as a staging page. 

Any pointers appreciated!

**Comment by pashidlos** [comment]
@openaudible why would you need to compare stage to production?
after deployment they should be the same
so compare new stage to old stage (current production) and see what was changed

**Reply by openaudible** [reply]
Maybe I'm missing terminology. Production might be 1.0, staging 1.1. When I'm happy I upgrade production to 1.1. 

I can push screenshots and see both sets of images. I just don't know how to set one as the baseline (prod) and compare it to the latest push (staged for production) 

When I view the images it only shows an image on the right, with no comparison image. (And I can approve the images, but that doesn't seem to do much.?)

Again, this is mostly when I or a dev makes a css change I want to see what got fixed/broken. 

Thanks!

**Reply by pashidlos** [reply]
@openaudible after you approve the image the next time you run the test is should already compare it to baseline
